homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -104,7 +104,10 @@ async def async_update(self) -> None:
 
     def _update_from_feed(self, feed_entry: GenericFeedEntry) -> None:
         """Update the internal state from the provided feed entry."""
-        self._attr_name = feed_entry.title
+        if feed_entry.properties and feed_entry.properties.get("name"):
+            self._attr_name = feed_entry.properties.get("name")
```

Comment:
```suggestion
        if feed_entry.properties and (name := feed_entry.properties.get("name")):
            self._attr_name = name
```

Reply:
What is the name doing here? It gives a ruff error:

`Error: homeassistant/components/geo_json_events/geo_location.py:107:39: F841 Local variable `name` is assigned to but never used`

Reply:
This makes us only get the name once, should be more efficient. Strange that it raises that error

Reply:
I will make it so, it works for me.